foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/generator/util/PythonGeneratorUtils.java

```diff
@@ -36,6 +37,7 @@ public static VelocityContext populateCommonPythonContext(VelocityContext veloci
         if (velocityContext == null) {
             velocityContext = new VelocityContext();
         }
+        rootArtifactId = generationContext.getRootArtifactId();
```

Comment:
I: This is done so that we can retrieve rootArtifactID in the metamodel package.